ckanext/who_afro/schemas/dataset.yaml

```diff
@@ -21,20 +21,37 @@ dataset_fields:
   form_snippet: markdown.html
   form_placeholder: eg. Some useful notes about the data
 
-- field_name: program_area
-  label: Program Area
-  preset: select
+- field_name: programme
```

Comment:
Did we discuss a way to pull out the programme or country lists so we don't have to make three changes everytime?

Reply:
Yes we can create a preset for them. This is actually simple change and Im happy to do that now :+1: